### DISCUSSION TITLE
How do I import a site into new Publii installation?

### DISCUSSION THREAD

**Discussion by lrhazi** [GetPublii/Publii#1506] [General] [answered]
I created a simple site in one machine, published it via sftp to web server.
How do I import it into a new machine where I just installed Publii?

Thanks!
Mohamed.

**Comment by bobmitro** [comment, marked as answer]
Create a website backup, and then add the new website on a new machine by choosing "**Install from backup**". 
More details: https://getpublii.com/docs/creating-your-website.html#creatingasitefromabackup

**Reply by lrhazi** [reply]
Thank you! 
but I can not import just the actual website files, as published from my other Publii installation, right?

**Reply by internettips** [reply]
@Irhazi

Publii uses various types of folders and files. The easiest, best, quickest way to achieve your goal is to do what @bobmitro outlined above. Takes only a few minutes.

**Reply by lrhazi** [reply]
sounds good. Thank you guys!
